cluster-proxy/src/main/java/com/sequenceiq/cloudbreak/clusterproxy/ClusterProxyEnablementService.java

```diff
@@ -0,0 +1,18 @@
+package com.sequenceiq.cloudbreak.clusterproxy;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+
+@Component
+public class ClusterProxyEnablementService {
+
+    @Inject
+    private ClusterProxyConfiguration clusterProxyConfiguration;
+
+    public boolean isClusterProxyApplicable(String cloudPlatform) {
+        return clusterProxyConfiguration.isClusterProxyIntegrationEnabled() && !CloudPlatform.MOCK.name().equals(cloudPlatform);
```

Comment:
instead of a hard coded value, could you use a configurable one and instead of a single item it could be a set/list so we can disable it on multiple platforms if necessary

Reply:
fair point. let me do that